components/tools/OmeroWeb/omeroweb/webclient/static/webclient/javascript/ome.tree.js

```diff
@@ -474,10 +474,10 @@ $(function() {
                 // Extra data needed for showing thumbs in centre panel
                 if (node.type === 'dataset' || node.type === 'orphaned' || node.type === 'tag') {
                     payload['sizeXYZ'] = true;
-                    payload['date'] = true;
-                    if (node.type !== 'tag') {
-                        payload['thumbVersion'] = true;
-                    }
+                    //payload['date'] = true;
+                    //if (node.type !== 'tag') {
+                    //    payload['thumbVersion'] = true;
+                    //}
```

Comment:
We need to keep ```payload['date'] = true``` (used in thumbnail list view in center panel).
Rest can be removed.